modules/rosa-adding-tags.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags_{context}"]
+= Adding tags to a machine pool
+
+You can add tags for compute nodes (also known as worker nodes) in a machine pool to introduce custom user tags for AWS resources that are generated when you provision your machine pool.
```

Comment:
Suggestion to replace parentheses with commas, since the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#general-text) suggests avoiding them in general text:
```suggestion
You can add tags for compute nodes, also known as worker nodes, in a machine pool to introduce custom user tags for AWS resources that are generated when you provision your machine pool.
```